.docker-compose.env

```diff
@@ -17,4 +17,4 @@ BROKER_URL=amqp://guest:guest@192.168.168.167:5672/
 REDIS_URL=redis://192.168.168.167:6379/1
 EMBER_DOMAIN=192.168.168.167
 
-#PYTHONUNBUFFERED=0 # This when set to 0 will allow print statements to be visible in the Docker logs
+PYTHONUNBUFFERED=0 # This when set to 0 will allow print statements to be visible in the Docker logs
```

Comment:
Why was this changed?

Reply:
This is the code equivalent of gum stuck to the bottom of your shoe.